lib/logstash/outputs/elasticsearch.rb

```diff
@@ -458,10 +458,8 @@ def submit(actions)
   public
   def flush(actions, teardown = false)
     begin
-      @logger.debug? and @logger.debug "Sending bulk of actions to client[#{@client_idx}]: #{@host[@client_idx]}"
       submit(actions)
     rescue => e
-      @logger.error "Got error to send bulk of actions to elasticsearch server at #{@host[@client_idx]} : #{e.message}"
```

Comment:
Well sometimes, the raise is not very helpful and cryptic to non ruby developers..so maybe we can keep this? and check for `host.nil?`

Agree with the first debug log line